### PR TITLE
Auth app - merge vue files

### PR DESCRIPTION
**Pull request by icaroharry**
### My PR doesn't concern a Jira task
Merging vue files into one and refactoring folder structure:
* Changed `users/create`  to `signup/utopian`
---
### **Test plan**

Check if the auth app is not broken by accessing localhost:8081/login

### CODE REVIEW

**Comment by nothingismagick** [comment]
Have you locally updated your version to `quasar-cli@0.17.22` ?

File: packages/auth/src/pages/login.vue [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+</script>
+
+<template lang="pug">
+q-layout.u-page-login
+  .row
+    .col-md-4.col-sm-12.col-xs-12.column.justify-center
+      .column.items-center.login-from
+        img.q-mb-xl(src="~assets/img/logo-black.svg")
+        q-btn.q-mb-sm(@click="startGithubLogin", icon="mdi-github-circle", color="white", text-color="black", label="$t('login.button.login')")
```
Comment (nothingismagick) [inline packages/auth/src/pages/login.vue]: `:label` is required otherwise translation won't hook.
It should be:
```
:label="$t('login.button.login')"
```

Comment (icaroharry) [reply packages/auth/src/pages/login.vue]: I'm implementing your i18n module within the auth app in a separate PR. This is just a temporary label so I remember to change it later.

**Comment by icaroharry** [comment]
- _it did not detect that I was already logged in:_ good catch. I'll work on that
- _if I visit the page directly (aka not sent to the page from 8080), the login button redirects me to localhost:8081/undefined:_ I'm aware of that. I'll change the redirect approach in a future PR as well.

File: packages/auth/src/pages/login.vue (hunk 1 above)
Comment (nothingismagick) [reply packages/auth/src/pages/login.vue]: no - it's about the : in front of the word label

**Pull request merged**